src/org/labkey/test/components/ui/samples/BulkCreateSamplesDialog.java

```diff
@@ -0,0 +1,155 @@
+package org.labkey.test.components.ui.samples;
+
+import org.labkey.test.BootstrapLocators;
+import org.labkey.test.Locator;
+import org.labkey.test.components.bootstrap.ModalDialog;
+import org.labkey.test.components.glassLibrary.components.FilteringReactSelect;
+import org.labkey.test.components.ui.EntityInsertPanel;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class BulkCreateSamplesDialog extends ModalDialog
```

Comment:
Sorry I didn't notice this earlier. This should be more generic. The panel that pops this dialog is for any kind of entity. I don't think it needs any functional changes, just a rename and a move to a different package:
`org.labkey.test.components.ui.EntityBulkInsertDialog`
Should also update the title passed to the `ModalDialogFinder` in the constructor. Based on the [product code](https://github.com/LabKey/labkey-ui-components/blob/master/packages/components/src/components/entities/EntityInsertPanel.tsx#L779), `"Bulk Creation of"` should do the trick.

Reply:
I think this is a good catch.  I'll do it.